README.md

```diff
@@ -116,6 +116,16 @@ helm install "${SKYWALKING_RELEASE_NAME}" ${REPO}/skywalking -n "${SKYWALKING_RE
   -f ./skywalking/values-my-es.yaml
 ```
 
+## Install SkyWalking with Satellite
+
+Enable the satellite as gateway, and set the satellite image tag.
+
+```shell script
+helm install "${SKYWALKING_RELEASE_NAME}" ${REPO}/skywalking -n "${SKYWALKING_RELEASE_NAMESPACE}" \
+  --set satellite.enabled=true \
+  --set satellite.image.tag=v0.3.0
+```
```

Comment:
We should add description about user should set service address to satellite instead of oap, right?

Reply:
been added, please take a look.